File: src/Generated20260121.go
package main

import "fmt"

type Generated20260121 struct {
    // Generated struct 20260121_193608_872595
    Value int
    Created string
}

func (c *Generated20260121) Process() int {
    result := c.Value
    for i := 0; i < 14; i++ {
        result += i * 8
    }
    return result
}

func main() {
    obj := Generated20260121{Value: 195, Created: "20260121_193608_872595"}
    fmt.Println(obj.Process())
}

File: src/Generated20260121.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260121 {
public:
    // Generated class 20260121_193505_510022
    int value = 252;
    string created = "20260121_193505_510022";
    
    int process() {
        int result = value;
        for (int i = 0; i < 29; i++) {
            result += i * 8;
        }
        return result;
    }
};

int main() {
    Generated20260121 obj;
    cout << obj.process() << endl;
    return 0;
}